README.md

```diff
@@ -1,6 +1,3 @@
-When first attempting to use this app, you must set your own personal database to use it with. To direct the app to that database set your environment variable by using, process.env.PGDATABASE = "Personal Server", while running any commands on this code. For example
-
-process.env.PGDATABASE = "Personal Server" npm run setup-dbs
-
-The above command will first set the server to "Personal Server" and then setup 2 databases on that server as instructed in ./db/setup.sql
+When first attempting to use this app, you must set your own personal database to use it with. To direct the app to that database set your environment variable by using by creating a file by the name ".env.{YOUR SERVER}". Once done in this file put PGDATABASE=nc_games. 
```

Comment:
The file names should be .env.test and .env.development instead of the name of the server. The file .env.test should include the name of the test database and the file for .env.development should include the name of the development database.